synapse/lib/stormlib/infosec.py

```diff
@@ -206,7 +242,7 @@ async def calculate(self, node, save=True, vers='3.1'):
                     if MPR == 'X': MPR = PR
                     if MS == 'X': MS = S
                     if MC == 'X': MC = C
-                    if MI == 'X': MC = I
+                    if MI == 'X': MC = _I
                     if MA == 'X': MC = A
```

Comment:
Is this actually supposed to potentially overwrite MC multiple times or is that copypasta?

Reply:
Nope, and that led to finding two other calc bugs :).  Updated.